### DISCUSSION TITLE
New Datasets for comparison

### DISCUSSION THREAD

**Discussion by aitor-martinez-seras** [megvii-research/FSSD_OoD_Detection#5] [Q&A] [answered]
Hello,

Are you planning to include more OoD datasets?

**Comment by andrehuang** [comment, marked as answer]
Hi,

Thanks for the question! Unfortunately, there is no current plan for including more datasets.

**Reply by andrehuang** [reply]
However, if you have implemented the methods on new OOD datasets, please let us know. We will see what we can do.
